apps/files_sharing/src/components/SharingEntry.vue

```diff
@@ -78,6 +78,11 @@ export default {
 			} else if (this.share.type === this.SHARE_TYPES.SHARE_TYPE_GUEST) {
 				title += ` (${t('files_sharing', 'guest')})`
 			}
+			if (!this.isShareOwner && this.share.ownerDisplayName) {
+				title += t('files_sharing', ' by {initiator}', {
```

Comment:
leading / trailing spaces in translations are very likely to break

```suggestion
				title += ' ' + t('files_sharing', 'by {initiator}', {
```